pkg/apis/certman/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,6 @@
 // +build !ignore_autogenerated
 
-// Code generated by operator-sdk. DO NOT EDIT.
+// Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
Just curious, what version of operator-sdk are you using?

Reply:
operator-sdk version: v0.8.1, commit: 33b3bfe10176f8647f5354516fff29dea42b6342

Yeah, I thought that was an odd diff too.